### PR TITLE
Adds clone support to mayactl cmd

### PR DESCRIPTION
**Pull request by ashishranjan738**
**What this PR does / why we need it**: This PR adds clone support to mayactl cli tool . By merging this PR volume can be cloned from mayactl cmd by providing sourcevol,volname and snapname.

**Special notes for your reviewer**: Merge this PR after merging this https://github.com/openebs/maya/pull/374

### CODE REVIEW

File: cmd/mayactl/app/command/volume_clone.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The OpenEBS Authors.
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume_clone.go]: nit : 2018 ?

Comment (ashishranjan738) [reply cmd/mayactl/app/command/volume_clone.go]: ok

**Comment by utkarshmani1997** [comment]
I see another PR with same issue, please close this.

**Comment by ashishranjan738** [comment]
its not the same pr it add cli code to hit clone api

**Comment by ashishranjan738** [comment]
will be raising a seperate PR for this.

**Pull request closed without merging**